cirq-core/cirq/devices/grid_device_metadata.py

```diff
@@ -169,4 +169,9 @@ def _json_dict_(self):
 
     @classmethod
     def _from_json_dict_(cls, qubit_pairs, gateset, gate_durations, all_qubits, **kwargs):
-        return cls(qubit_pairs, gateset, dict(gate_durations), all_qubits)
+        return cls(
+            qubit_pairs,
+            gateset,
+            None if gate_durations is None else dict(gate_durations),
+            all_qubits,
+        )
```

Comment:
Why add this `None if gate durations` ?

Reply:
Non-default arguments can't be after default arguments. Will shifting the order of method parameters here break JSON backward compatibility?